CommonVoice-Data/data/gutenberg/29251.txt

```diff
@@ -1,2 +0,0 @@
-L'orthographe et la ponctuation sont conformes à l'original.
```

Comment:
it's not empty

Reply:
It's one line. It's not a book. It's not significant nor representative. It's clearly buggy. It's part of the tail of `ls -l`. Should I really argue?

Reply:
it's still a sentence that can be spoken :)